### PR TITLE
Update Diplomacy,Trade,Nations.json

### PR DESCRIPTION
**Pull request by VoyteeG**
Few translations in Polish added.

### CODE REVIEW

**Comment by VoyteeG** [comment]
Please, let me know what's gone wrong this time. Also instruction how to fix it.
Thanks

**Comment by yairm210** [comment]
You probably forgot a closing " somewhere

File: android/assets/jsons/Translations/Diplomacy,Trade,Nations.json [hunk 1]
```diff
@@ -545,6 +573,7 @@
 	"Please don't settle new cities near us.":{
 		Italian:"Smetti di fondare città vicino a noi."
 		French:"Veuillez ne pas construire de villes proches de nous."
+		Polish:"Proszę nie osiedlaj się w pobliżu naszych miast.
```
Comment (yairm210) [inline android/assets/jsons/Translations/Diplomacy,Trade,Nations.json]: Here, maybe there are other places as well

**Comment by VoyteeG** [comment]
If I made changes directly on main branch, can I reopen this file and fix the problem?
If yes, where and how?
Thank you

**Comment by VoyteeG** [comment]
I found the way. Thanks

**Pull request merged**